.gitignore

```diff
@@ -0,0 +1 @@
+debug.log
```

Comment:
great job

you may use ***.log** to automatically include any other log file there may be